src/microbe_stage/MicrobeHUD.cs

```diff
@@ -258,6 +258,11 @@ public override bool GetCurrentSpeedMode()
         return stage.WorldSimulation.WorldTimeScale > 1;
     }
 
+    public override bool IsHeatViewAvailable()
```

Comment:
Adding this new method means that this line: `bottomLeftBar.HeatViewAvailable = organelles.HeatCollection > 0;` in `UpdateAbilitiesHotBar` should also be updated to use this.